packages/loctool/test/testfiles/xliff_webOS/app2/ko-KR.xliff

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="2.0" srcLang="en-KR" trgLang="ko-KR" xmlns:l="http://ilib-js.com/loctool">
```

Comment:
Please add the namespace to ensure proper parsing and validation of metadata elements.
ex) 
<xliff xmlns="urn:oasis:names:tc:xliff:document:2.0" version="2.0"
       xmlns:mda="urn:oasis:names:tc:xliff:metadata:2.0"
       srcLang="en-KR" trgLang="ko-KR">

Reply:
I updated it.